### DISCUSSION TITLE
Installing jaxlib release v0.4.13 on Windows

### DISCUSSION THREAD

**Discussion by Kate-Lyndegaard** [jax-ml/jax#16628] [Q&A]
Hi,

I tried running `pip install jaxlib` to install [jaxlib release v0.4.13](https://github.com/google/jax/releases/tag/jaxlib-v0.4.13) on my Windows 10 machine. I have Python3.8 installed. I get the following:

ERROR: Could not find a version that satisfies the requirement jaxlib==0.4.13 (from versions: none)
ERROR: No matching distribution found for jaxlib==0.4.13

Do you have any suggestions?

**Comment by jakevdp** [comment]
Thanks for the question! Note that windows jaxlib support is currently [experimental](https://github.com/google/jax#pip-installation-cpu), and only available in jaxlib version 0.4.13. Additionally, jaxlib wheels for windows are only available for Python 3.9, 3.10, and 3.11 (see https://pypi.org/project/jaxlib/#files). Is it possible for you to update your Python version?

**Comment by Kate-Lyndegaard** [comment]
Hi Jake,

Sorry, I did not see that. I can update to 3.9. Thanks:-)

Kind regards,
Kate
